PocketCastsTests/UnitTests.xctestplan

```diff
@@ -13,7 +13,65 @@
     }
   ],
   "defaultOptions" : {
-    "codeCoverage" : false,
+    "codeCoverage" : {
```

Comment:
What do you think about:

1. Keeping `podcasts`
2. Keeping `Pocket Casts Watch App`
3. Removing the rest
4. Adding `PocketCastsDataModel`
5. Adding `PocketCastsServer`
6. Adding `PocketCastsUtil`

The reasoning for that is:

1. I feel it's easier to follow code coverage for a few targets. If we have too many of them, it's hard to follow — because it's too much
2. `DataModel`,  `Server` and `Utils` are probably the targets in which tests are really important, given that there's no UI and it should be easier to test (well, a lot of it in practice it's not but that's another story)

Let me know what do you think.

Reply:
> Adding PocketCastsDataModel
Adding PocketCastsServer
Adding PocketCastsUtil

Good catch I missed these 👍

> Removing the rest

It seem's there [an issue](https://developer.apple.com/forums/thread/133517) with getting a packages test coverage to display when selecting specific classes to get coverage for. At present to show a packages test coverage you need to get coverage for all targets within that scheme. (Useful to know for when we do this for Tumblr).

I think it's still worth doing as you still get the side bar coverage and could select a specific report if it's of interest to you. Although it does mean you need to disregard the total figure xcode provides as it includes Kingfisher and test packages.

Here's a screenshot.

![image](https://github.com/Automattic/pocket-casts-ios/assets/18557720/67717362-b9c6-4efc-97ee-ca515b5013f2)